packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -81,6 +122,38 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
       })
       .next(() => results);
   }
+
+  getNextDocumentChanges(
```

Comment:
I think getNewDocumentChanges() might convey better that this returns changes since the last call. Don't feel super strongly though.

Reply:
I am fine with either. Renamed to `getNewDocumentChanges`.